src/formatDate.js

```diff
@@ -8,7 +8,26 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const toSeparator = toFormat.pop();
```

Comment:
Similarly, using `Array.prototype.pop` on `toFormat` will mutate the input array. It's important to avoid side effects on inputs unless explicitly intended.